securedrop/tests/utils/env.py

```diff
@@ -59,6 +60,9 @@ def setup():
 
 
 def teardown():
+    for t in threading.enumerate():
```

Comment:
good idea. I added a comment, possibly with typos... I'd be grateful if you could verify ;-)